File: blog-admin/src/type/album.ts
export interface AlbumVO {
    id: number,
    albumName: string,
    albumDesc: string,
    albumCover: string,
    userId: number,
    nickname: string,
    createTime: string,
    isDel: number
}

export interface AlbumDTO{
    id: number,
    albumName: string,
    albumDesc: string,
    albumCover: string,
}

export interface Album{
    id: number,
    albumName: string,
    albumDesc: string,
    albumCover: string,
    userId: number,
    nickname: string,
    createTime: string,
    photoList: Photo[]
}

export interface SimpleAlbumDTO{
    id: number,
    albumName: string,
}

export interface Photo{
    id: number,
    photoName: string,
    photoUrl: string,
    albumId: number,
    userId: number,
    nickname: string,
    createTime: string,
    isDel: number
}

export interface PhotoDTO{
    id: number,
    photoName: string,
    photoUrl: string,
    albumId: number,
}

export interface LocalPhotoTempData{
    id: number,
    photoName: string,
    photoUrl: string,
    albumId: number,
    imageFile: string,
    fileName: string
}

File: yelingzi-blog/src/main/java/com/yeling/album/domian/entity/Album.java
package com.yeling.album.domian.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Album {

    private Integer id;

    private String albumName;

    private String albumDesc;

    private String albumCover;

    private Integer userId;

    private String nickname;

    private String createTime;

    private Integer isDel;

}
